setup.py

```diff
@@ -1,82 +1,64 @@
 import os
-import re
 import sys
 import platform
 import subprocess
 
-from setuptools import setup, find_packages, Extension
+from setuptools import setup, Extension
 from setuptools.command.build_ext import build_ext
-from distutils.version import LooseVersion
 
 
 class CMakeExtension(Extension):
     def __init__(self, name):
         Extension.__init__(self, name, sources=[])
 
+
 class CMakeBuild(build_ext):
     def build_extension(self, ext):
-
         extdir = os.path.abspath(os.path.dirname(self.get_ext_fullpath(ext.name)))
 
         debug = int(os.environ.get("DEBUG", 0)) if self.debug is None else self.debug
-        cfg = 'Debug' if self.debug else 'Release'
-        print('building CMake extension in %s configuration' % cfg)
+        cfg = "Debug" if self.debug else "Release"
+        print("building CMake extension in %s configuration" % cfg)
 
         cmake_args = [
```

Comment:
When only using the normal compilation, this happens

`subprocess.CalledProcessError: Command '['cmake', '--build', '.', '--target', 'dpsimpy', '--', '-j4']' returned non-zero exit status 2.`

Should we enable `-DFETCH_PYBIND` here or is something only related to my system?

Reply:
Good question.. I would say its depending on the environment where you build DPsim and which dependencies are available via the systems package manager.

I think we should not change the flags in this PR. If its an issue, its unrelated to the original goal of this PR and hence should be handled in a separate one.